.travis.yml

```diff
@@ -3,6 +3,8 @@ version: ~> 1.0
 import:
   - silverstripe/silverstripe-travis-shared:config/provision/self-jobs-range.yml
 
+# Require simple theme for a few functional tests, otherwise the 'Welcome to Silverstripe' no theme fallback template
+# is shown which will break the tests
 env:
   global:
-    - REQUIRE_INSTALLER="*"
+    - REQUIRE_EXTRA="silverstripe-themes/simple:^3"
```

Comment:
Looks like requiring the simple theme without requiring installer fails the build. I'm thinking maybe we should remove this kind of statement on current build that have it because it probably doesn't do anything any way.

Reply:
Updated